### DISCUSSION TITLE
rn app运行不起来

### DISCUSSION THREAD

**Discussion by Newbit13** [NervJS/taro#10856] [Q & A]
![a1cb798438d24fcfe0fa6127999c5b2](https://user-images.githubusercontent.com/17379717/145242973-6c3b75d4-fe30-4275-ad7d-abd3f422cae5.jpg)

这有解决思路吗，我用官网提供的taro-native-shell，然后@tarojs/cli版本是3.3.16，模板是拉取github上mobx模板，运行起来就是这样了

**Comment by fangjunai** [comment]
我也碰到这个为情况了。 怎么解决？

**Comment by zhiqingchen** [comment]
使用 react-native 模板

**Reply by fangjunai** [reply]
> 使用 react-native 模板
已经选择 react-native 模板 还是不行啊。 还有没有其他问题？

**Comment by fangjunai** [comment]
taro-native-shell 的版本不匹配引起的，最新的是：[https://github.com/NervJS/taro-native-shell/tree/0.66.0](https://github.com/NervJS/taro-native-shell/tree/0.66.0)
